catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/handlers/CqlTransformHandler.java

```diff
@@ -141,6 +147,14 @@ public Object handle(Request request, Response response) throws Exception {
 
     CqlQueryResponse cqlQueryResponse = util.executeCqlQuery(cqlRequest);
 
+    Object schema = queryResponseTransformer.getProperty("schema");
+
+    if (queryResponseTransformer.getProperty("mime-type").toString().equals("[text/csv]")) {
```

Comment:
The mime-type is a list; might be better to check if it contains `text/csv` instead of using the toString method